Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo ()
+        {
+            string userFirstName = UserName(message: "Enter your first name: ");
+            string userSurname = UserName(message: "Enter your surname: ");
+            int userAge = UserAge(message: "Enter your age: ");
+            float userHeight = UserHeightWeight(message: "Enter your height in cm: ");
+            float userWeight = UserHeightWeight(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            double bmi = weight / Math.Pow(height, 2);
+            return Convert.ToSingle(bmi);
```

Comment:
You're doing quite heavy stuff here which you don't need to do.
Consider using float bmi = weight / (height * height) to spare the unneccessary extra work.